pkg/devfile/adapters/docker/adapter.go

```diff
@@ -35,6 +35,8 @@ func (d Adapter) Push(parameters common.PushParameters) error {
 
 func (k Adapter) Build(parameters common.BuildParameters) error { return nil }
 
+func (k Adapter) Deploy(parameters common.DeployParameters) error { return nil }
+
```

Comment:
Just gonna write here as a TODO, to throw an error to user to say that this command doesn't work with docker